frontend/conf/routes

```diff
@@ -20,10 +20,14 @@ GET            /analytics-off                         controllers.Testing.analyt
 GET            /join/staff                            controllers.Joiner.staff
 GET            /join/staff/enter-details              controllers.Joiner.enterStaffDetails
 GET            /join/friend/enter-details             controllers.Joiner.enterFriendDetails
+
+GET            /join/monthly-contribution/enter-details controllers.Joiner.enterMonthlyContributionsDetails(countryGroup: CountryGroup ?= CountryGroup.UK)
```

Comment:
As discussed remove /join from the url as we don't want to confuse monthly contributions with the existing concept of 'Joining the Guardian'